### PR TITLE
Feature GRTLV-329 - Implement great design system header

### PR DESCRIPTION
**Pull request by boxadesign**
DO NOT MERGE

## What
Implement the domestic header (includes microsites) from the Great Design System.

There is an update to the look and feel. The functionality should be identical.

The main way of testing is to navigate around as many different parts of the Great website to check that the header works consistently. Due to different parts of the website loading different stylesheets there could be bugs from style conflicts. Also check the whole page as styles have been removed/moved around.

## Why
To further decouple front end code from great-cms

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GRTLV-329
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### CODE REVIEW

**Review by stuart-mindt — APPROVED** [review]
Looks good

File: core/context_processors.py [hunk 1]
```diff
@@ -119,3 +119,132 @@ def services_home_links(request):
     return {
         'international_home_link': {'url': reverse_lazy('index'), 'label': _('great.gov.uk international')},
     }
+
+
+def domestic_header(request):
```
Comment (FoxJamie16) [inline core/context_processors.py:124]: I pulled down this branch and then merged from develop which just had @stuart-mindt 's international branch merged, and the domestic site is now showing the international header due to shared context names. For the footer I solved this by declaring as : 
`def domestic_footer(request):
    return { 'domesticFooterContext': {
        'isInternational': False,`
which could be suitable here? 
Screenshot 2024-10-25 at 11 30 31

Comment (boxadesign) [reply core/context_processors.py:124]: Yep I spotted that, thats all sorted now

**Review by FoxJamie16 — APPROVED** [review]
LGTM

**Pull request merged**